tests/integration_tests/test_fulltext_parser_config.py

```
@@ -113,8 +114,8 @@ def _test_default_parser(self, client):
 
         assert collection is not None
 
-        # Verify default parser (ik) is used
-        table_name = f"c$v1${test_collection_name}"
+        # Verify default parser (ik) is used  
+        table_name = CollectionNames.table_name(test_collection_name)
```

Ruff (0.14.10)

Undefined name `CollectionNames`

(F821)